components/brave_rewards/browser/database/publisher_info_database.cc

```diff
@@ -1239,6 +1239,21 @@ bool PublisherInfoDatabase::DeleteUnblindedToken(
   return unblinded_token_->DeleteRecord(&GetDB(), id_list);
 }
 
+bool PublisherInfoDatabase::DeleteUnblindedTokensForPromotion(
+    const std::string& promotion_id) {
+  DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
+
+  bool initialized = Init();
+  DCHECK(initialized);
+
+  if (!initialized) {
+    return false;
+  }
+
+  return DatabaseUnblindedToken::DeleteRecordsForPromotion(&GetDB(),
```

Comment:
nit: can you please put each param in separate like

```
return DatabaseUnblindedToken::DeleteRecordsForPromotion(
       &GetDB(),
       promotion_id);
```

Reply:
Done